tests/containerfiles/lbi/usr/share/containers/systemd/jboss-webserver-5.image

```diff
@@ -0,0 +1,3 @@
+[Image]
+Image=registry.redhat.io/jboss-webserver-5/jws5-rhel8-operator:latest
+AuthFile=/root/auth.json
```

Comment:
Is this actually used?

Reply:
nope, added a comment to explain that it's meant to represent a quadlet image that is not logically bound.